### PR TITLE
Alert and Button Components

### PR DESCRIPTION
**Pull request by SRUJANA-PENUGONDA13**
### Alert
 Added basic alerts

### Button
Added different types of buttons

- Primary
- Link
- Icon Button
- Floating Action Button

### CODE REVIEW

**Review by sanjay-xdr — COMMENTED** [review]
add more than two types of button refer to bootstrap or any other similar library 
format your html code properly

File: css/components/alert.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.alert {
+  border-radius: 10px;
+  font-size: 1.5rem;
+  padding: 1rem;
+}
+.alert-primary {
+  background-color: var(--primary-alert);
+  border: 1px solid #2a7fff;
```
Comment (devdeadviz) [inline css/components/alert.css]: Use variables for this color #2a7fff also.

File: css/components/alert.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  color: var(--primary--color);
+}
+.alert-secondary {
+  background-color: var(--secondary-alert);
+  border: 1px solid #919191;
```
Comment (devdeadviz) [inline css/components/alert.css]: same here

**Review by devdeadviz — APPROVED** [review]
Added some suggestions, have a look at it.

**Comment by SRUJANA-PENUGONDA13** [comment]
Updated suggested changes for colour variables and will add more buttons in next version of component library

**Pull request merged**